openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -426,10 +426,12 @@ When the grant type `authorization_code` is used, it is RECOMMENDED to use PKCE
 
 An Authorization Request is an OAuth 2.0 Authorization Request as defined in Section 4.1.1 of [@!RFC6749], which requests that access be granted to the Credential Endpoint, as defined in (#credential-endpoint).
 
-There are two possible ways to request issuance of a specific Credential type in an Authorization Request. One way is to use the `authorization_details` request parameter, as defined in [@!RFC9396], with one or more authorization details objects of type `openid_credential`, per (#authorization-details). The other is through the use of scopes as defined in (#credential-request-using-type-specific-scope).
+There are two possible methods for requesting the issuance of a specific Credential type in an Authorization Request. The first method involves using the `authorization_details` request parameter, as defined in [@!RFC9396], containing one or more authorization details of type `openid_credential`, as specified in (#authorization-details). The second method utilizes scopes, as outlined in (#credential-request-using-type-specific-scope). A Client MUST use one of these methods but MUST NOT use both simultaneously.
```

Comment:
I suggest just clarifying that to make clear that we're not trying to say, e.g., that you can't request a credential using RAR at the same time as requesting (say) access to an API using scopes (RAR does allow authorisation details & scopes to be used in the same request in general, https://datatracker.ietf.org/doc/html/rfc9396#section-3.1 ):

```suggestion
There are two possible methods for requesting the issuance of a specific Credential type in an Authorization Request. The first method involves using the `authorization_details` request parameter, as defined in [@!RFC9396], containing one or more authorization details of type `openid_credential`, as specified in (#authorization-details). The second method utilizes scopes, as outlined in (#credential-request-using-type-specific-scope). A Client MUST use one of these methods to request Credentials but MUST NOT use both simultaneously.
```